smart-contracts/contracts/mixins/MixinRefunds.sol

```diff
@@ -37,14 +36,13 @@ contract MixinRefunds is
 
   event RefundPenaltyChanged(
     uint freeTrialLength,
-    uint refundPenaltyNumerator,
-    uint refundPenaltyDenominator
+    uint refundPenaltyBasisPoints
   );
 
   function initialize() public
   {
-    refundPenaltyNumerator = 1;
-    refundPenaltyDenominator = 10;
+    // default to 10%
+    refundPenaltyBasisPoints = 1000;
```

Comment:
So 1000 basis points is the same as 0.1, correct?

Reply:
yup